DEV_NOTES.md

```diff
@@ -141,6 +141,241 @@ display name somehow. I'm not sure how. If someone figures it out, please let me
 updating the Contact `DISPLAY_NAME` directly but it does not work. Setting a name row as default 
 also does not affect the Contact `DISPLAY_NAME`.
 
+### Contacts; ID vs LOOKUP_KEY
```

Comment:
Researching the behaviors from API levels 19 to 31 and documenting it was the hardest part of this whole thing. Writing the code was a breeze.